packages/pointer_interceptor/example/lib/src/shim/dart_ui_fake.dart

```diff
@@ -22,7 +22,7 @@ class webOnlyAssetManager {
   /// Shim for getAssetUrl.
   /// https://github.com/flutter/engine/blob/master/lib/web_ui/lib/src/engine/assets.dart#L45
   static String getAssetUrl(String asset) {
-    return null;
+    return '';
```

Comment:
This is fine. The Shim code should actually never be called, this exists here to appease the analyzer about web-only methods in the dart:ui library.